app/Helpers/Models/LegiScan/SessionYear.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Helpers\Models\LegiScan;
```

Comment:
... Why is this namespace so complicated?? ... What value is that adding?

Also, I never said to NOT make a Facade. I just said to not make a Facade the hardest way you can make a facade...

Can this just be `App\Helpers\SessionYear`?

Can there be an `App\Facades\SessionYear` that just says

```
return app(SessionYear::class);
```

And then everywhere else we just say

```
use App\Facades\SessionsYear
...
SesssionYear::getAll()
```

or

```
{ \App\FacadesSessionYear::getAll()}
``

in views?...

Reply:
I was making the helper keep a consistent nesting as everything else.

I might need to do a quick Zoom for this one.

Reply:
... or a Slack huddle.

Reply:
I'm going to just set it per your initial comment.

Reply:
Done